vault/dendron.topic.refs.md

```diff
@@ -54,6 +54,12 @@ A note reference will include the entire contents of a note within another note.
 
 ![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/refs.block.gif)
 
+Note that references also respect the wildcard operator. The following example will include all child pages of sample. 
```

Comment:
can we add this as its own section? eg. `Block Reference with Wildcard`

Reply:
I thought about this but isn't the wildcard operator usable in the three other forms of using refs? I.e. you can use the wildcard operator in note refs, block refs and block range refs. I didn't think this is actually a 4th "type".

Reply:
that's fair